clearcut_detection_backend/test/download.py

```diff
@@ -0,0 +1,287 @@
+import os
```

Comment:
do you need to copy all funcitonaliity? or just can import the function from different module?